packages/bits/demo/src/components/demo/tabgroup/tabgroup-docs/tabgroup-docs.example.component.html

```diff
@@ -229,7 +229,9 @@ <h2>
 <p>
     <nui-message type="warning" allowDismiss="true">
         Because this example is in the iframe you won't see the route changes. To watch routes change in browser's address bar please navigate
-        to <a href="./#/tabgroup" target="_blank">this exmaples page</a>. The link opens in a new tab.
+        to <a href="./#/tabgroup" target="_blank">this exmaples page</a>. The link opens in a new tab. <br>
```

Comment:
A pre-existing typo. Might as well fix it.

```suggestion
        to <a href="./#/tabgroup" target="_blank">this examples page</a>. The link opens in a new tab. <br>
```